doc/api_assets/sh.css

```diff
@@ -1,25 +1,23 @@
-.sh_sourceCode {
+.hljs {
```

Comment:
The file should probably be renamed

Reply:
What would you recommend?

- `hl.css`
- `hljs.css`
- `code.css`
- `default.css`
- `syntax.css`
- `node-syntax-light.css`

I had trouble choosing, tbh.

Reply:
`hljs.css`